lib/rspec/support/ruby_features.rb

```diff
@@ -45,7 +45,7 @@ def mri?
     # Provides query methods for ruby features that differ among
     # implementations.
     module RubyFeatures
-      module_function
+    module_function
```

Comment:
Yep see [`Module#module_function`](http://ruby-doc.org/core-2.2.2/Module.html#method-i-module_function). It's pretty handy when you have a set of utility functions that you want to reference but do not want to increase an object's public API.